x/concentrated-liquidity/incentives_test.go

```diff
@@ -3526,6 +3519,29 @@ func (s *KeeperTestSuite) TestQueryAndClaimAllIncentives() {
 				}
 				s.Require().Equal(expectedCoins, amountClaimed)
 				s.Require().Equal(sdk.Coins(nil), amountForfeited)
+
+				// Sum of growthInside
+				totalGrowthInside := sdk.NewDecCoins()
+				for _, coin := range tc.growthInside {
+					totalGrowthInside = totalGrowthInside.Add(coin...)
+				}
+
+				// sum of  growthOutside
+				totalGrowthOutside := sdk.NewDecCoins()
+				for _, coin := range tc.growthOutside {
+					totalGrowthOutside = totalGrowthOutside.Add(coin...)
+				}
+
+				// sum of uptimeAccumulatorGrowth
+				uptimeAccumulatorGrowth := sdk.NewDecCoins()
+				for _, coin := range newUptimeAccumValues {
+					uptimeAccumulatorGrowth = uptimeAccumulatorGrowth.Add(coin...)
+				}
+
+				// Calculate the totalSum, should be tc.growthInside + tc.growthOutside
+				expectedTotal := totalGrowthInside.Add(totalGrowthOutside...)
+
+				s.Require().Equal(expectedTotal, uptimeAccumulatorGrowth)
```

Comment:
Could you elaborate on what the goal of this is, please?

What I would expect for testing that uptime accumulators are updated is that the block time from position creation time to collection time is increased. As a result, when calling claim incentives, `updateUptimeAccumulatorsToNow` ends up distributing per every uptime that is satisfied

Reply:
I think this test case can be used as a reference: https://github.com/osmosis-labs/osmosis/blob/af3a17400eef846fc0fafe8ce44b9d5029633b45/x/concentrated-liquidity/incentives_test.go#L2920-L3008

I think that porting some of the logic from there into a direct unit test for `claimAllIncentivesForPosition` would be sufficient for this issue.

The goal is to have strict coverage so that if I were to remove [this](https://github.com/osmosis-labs/osmosis/blob/af3a17400eef846fc0fafe8ce44b9d5029633b45/x/concentrated-liquidity/incentives.go#L838-L841), the direct unit tests would break.

Currently only `TestFunctional_ClaimIncentices_LiquidityChange_VaryingTime` breaks which is at another level of abstraction

Reply:
updated https://github.com/osmosis-labs/osmosis/pull/5290/commits/516cfbdbfaa41521cf9f39c51ffd1f5453dc0db6

thank you for the advice